src/PaymentBundle/Resources/docs/reference/configuration.rst

```diff
@@ -117,7 +117,7 @@ Full Configuration Options
         selector: sonata.payment.selector.simple
 
         # service which generate the correct order and invoice number
-        generator: sonata.payment.generator.mysql
+        generator: sonata.payment.generator.mysql # or sonata.payment.generator.postgres
```

Comment:
No limiting sounds Not that great, but expose our implementation in an additional info() node